docker-compose.yml

```diff
@@ -49,9 +49,9 @@ services:
     restart: always
     network_mode: bridge
     expose:
-      - 1033
+      - 3033
     ports:
-      - "1033:5432"
+      - "3033:5432"
```

Comment:
If you change that here, you also need to change it in GraphQlApiTest annotation

Reply:
ok too late xD but that can also be done later

Reply:
That's what I did. Check in the annotation file ;)